### DISCUSSION TITLE
Lack of Markdown Support in User Message Input

### DISCUSSION THREAD

**Discussion by guoruqiang** [open-webui/open-webui#4472] [Ideas]
#### Description
In the current version, users are unable to send messages with Markdown formatting. This limitation makes it inconvenient to format messages, as users cannot utilize common text formats such as code blocks and LaTeX equations to enhance message readability.

#### Desired Feature
I would like to see support for Markdown formatting in the message input box, allowing users to easily format their messages. For example:
- Use ``` to designate code block sections.
- Support for recognizing LaTeX equations.

Such an improvement would greatly enhance the user experience, enabling us to communicate information more effectively.

#### Steps to Reproduce
1. Open the application.
2. Attempt to send a message using Markdown formatting in the input box.
3. Notice that the message is sent without the expected formatting.

Thank you for considering this! I look forward to your response and any relevant improvements.

**Comment by tjbck** [comment]
Added to dev!

**Comment by praj88** [comment]
is this feature of supporting markdown response from llm supported
